source/adapters/hip/CMakeLists.txt

```diff
@@ -11,12 +11,22 @@ set(UR_HIP_PLATFORM "AMD" CACHE STRING "UR HIP platform, AMD or NVIDIA")
 # Set default ROCm installation directory
 set(UR_HIP_ROCM_DIR "/opt/rocm" CACHE STRING "ROCm installation dir")
 
-set(UR_HIP_INCLUDE_DIR "${UR_HIP_ROCM_DIR}/include")
-
-set(UR_HIP_HSA_INCLUDE_DIR "${UR_HIP_ROCM_DIR}/hsa/include")
-
-# Set HIP lib dir
-set(UR_HIP_LIB_DIR "${UR_HIP_ROCM_DIR}/lib")
+# Allow custom location of HIP/HSA include and HIP library directories
+if("${SYCL_BUILD_PI_HIP_INCLUDE_DIR}" STREQUAL "")
+  set(UR_HIP_INCLUDE_DIR "${UR_HIP_ROCM_DIR}/include")
+else()
+  set(UR_HIP_INCLUDE_DIR "${SYCL_BUILD_PI_HIP_INCLUDE_DIR}")
```

Comment:
I think we should avoid using `SYCL_` prefixed CMake variables in UR. Instead we should define these `UR_HIP_` variables as cache variables which can then be set by the SYCL CMake when the HIP adapter is enabled.

We do something similar for specifying the OpenCL-ICD-Loader tot he OpenCL adatper [here](https://github.com/oneapi-src/unified-runtime/blob/main/source/adapters/opencl/CMakeLists.txt#L9) and in intel/llvm [here](https://github.com/intel/llvm/blob/sycl/sycl/plugins/unified_runtime/CMakeLists.txt#L33-L34).